tests/compilers/test_ast.py

```diff
@@ -664,3 +664,21 @@ def test_ast_bad_yield_from():
 def test_eval_generator_with_return():
     """Ensure generators with a return statement works."""
     can_eval("(fn [] (yield 1) (yield 2) (return))")
+
+
+def test_docsting_imports():
+    """Confirm that docstrings come before import statements in AST output."""
+    hy_tree = import_buffer_to_hst('"hello world"\n(print (first [1 2]))')
+    hy_ast = hy_compile(hy_tree, 'test')
+    assert hy_ast.body[0].value.s == 'hello world'
```

Comment:
I think it would be a better test to add a module docstring to `resources/pydemo.hy` and check for it in `test_hy2py`.

Reply:
You fine leaving that test in there, though?  I like its "directness".

Reply:
It uses an internal function (`import_buffer_to_hst`), and it relies on the structure of Python AST, to test for a bug that's readily visible in user space. So I'd say we should get rid of it if we don't need it.

Reply:
Looks like the tests in `test_ast.py` are using `can_compile` instead; I'll make the other tests here use that as well.

Otherwise, I can replace this one with an assertion to [`test_hy2py`](https://github.com/hylang/hy/blob/master/tests/test_hy2py.py#L25) that checks `m.__doc__`.